frontend/src/app/modules/common/header/quick-add-menu.sass

```diff
@@ -2,33 +2,19 @@
   @include breakpoint(680px down)
     display: none
 
-  &--button
+  &--icon::before
     display: flex
     justify-content: center
     align-items: center
-    position: relative
-    color: var(--button--alt-highlight-font-color)
-    font-size: 0.8rem
+    border: 2px solid var(--header-item-font-color)
+    border-radius: 50%
+    width: 26px
+    height: 26px
+    padding: 1px 8px 0 8px
+    background: var(--button--alt-highlight-background-color)
+    border-color: var(--button--alt-highlight-background-color)
 
-    > *
-      z-index: 1
-
-    &::after
-      display: block
-      background: var(--button--alt-highlight-background-color)
-      border-width: 2px
-      border-style: solid
-      border-color: var(--button--alt-highlight-background-color)
-      border-radius: 50%
-      padding: 0.25rem
-      width: 26px
-      height: 26px
-      content: ''
-      position: absolute
-
-    &:hover
-      > *
-        opacity: 0.9
-
-      &::after
-        opacity: 0.8
+  &:hover
+    .op-quick-add-menu--icon::before
```

Comment:
You can shorten this with `&:hover #{&}--icon::before` as a selector, though I'm not sure if it's much more readable.

Reply:
You can also do

```
&:hover &
  &--icon:before
```

which matches a bit more the BEM syntax we usually use without interpolation

Reply:
That's cool I hadn't thought of that yet :+1:

Reply:
Nice, I learned something today, I believe. Thank you both